index.bs

```diff
@@ -520,6 +534,29 @@ noise suppression using Recurrent Neural Network such as [[RNNoise]] for
 suppressing background dynamic noise like baby cry or dog barking to improve
 audio experiences in video conferences.
 
+### Speech Recognition ### {#usecase-speech-recognition}
+
+Speech recognition, also known as speech to text, enables recognition and
+translation of spoken language into text. Example applications of speech
+recognition include transcription, automatic translation, multimodal interaction,
+real-time captioning and virtual assistants. Speech recognition improves
+accessibility of auditory content and makes it possible to interact with such
+content in a privacy-preserving manner in a textual form. Examples of common
+use cases include watching videos or participating online meetings using
```

Comment:
nit: _"...participating in online meetings..."_

Reply:
Fixed by https://github.com/webmachinelearning/webnn/pull/507/commits/1d0f5c04967fdd4cec6c3da7f942d8eeb1265e1f